src/tracer.js

```diff
@@ -31,9 +33,19 @@ let currRunner = null;
 const pendingEvents = new Map();
 
 /**
- * The timeout to send for send operations (both sync and asyc)
+ * The timeout to send for send operations (both sync and async)
  */
-const sendTimeoutMiliseconds = 1000;
+const sendTimeoutMilliseconds = 1000;
+
+/**
+ * Session for the post requests to the collector
+ */
+const session = axios.create({
+    timeout: sendTimeoutMilliseconds,
+    httpAgent: new http.Agent({ keepAlive: true }),
+    httpsAgent: new https.Agent({ keepAlive: true }),
+});
+module.exports.sessionPost = session.post;
```

Comment:
why are you exporting this? if this is for the UT you should mock the result of axios.create instead